.travis.yml

```diff
@@ -37,3 +37,5 @@ install:
 script:
   - nimble install -y
   - nimble test
+  - nimble benchmark
```

Comment:
what's the purpose of running benchmark on CI setup? neither travis nor appveyor provide stable hardware so numbers will be meaningless, and benchmarks tend to take a while so it will slow down every PR / build roundtrip..